src/components/rating/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react';
+import { Box } from '@chakra-ui/react';
+
+export type RatingProps = {
+  rating: number;
+  size: 'large' | 'small';
+};
+
+export const Rating: FC<RatingProps> = ({ rating, size }) => {
+  return (
+    <Box
+      aria-label={`Rating is ${rating} out of 5`}
+      display="inline-block"
+      fontSize={size === 'large' ? 'md' : 'sm'}
+      fontFamily="makeItSans"
+      lineHeight="xs"
+      css={{
+        '--rating': rating.toString(),
+        '--percent': 'calc((var(--rating) - 0.16) / 5 * 100%)',
+        '--star-color': 'colors.orange.300',
```

Comment:
This was grey.900 in v2, is this an intentional change?

Reply:
Yes, with all the changes now, we need to pass the orange as a background color for the stars

Reply:
@Averethel 

**For more context:**
We had the rating in color gray but it was decided to change it to orange so time ago -> https://smg-au.atlassian.net/browse/DM-3655

When I branched from the stack branch it was not updated with this change. Therefore, we saw here the gray color.